### DISCUSSION TITLE
Accuracy Heatmap for osu!catch

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29920] [Ideas]
I like the ability to view my aim in more detail in !std, and i think it would also make sense if would be possible to see in !ctb.
Since taiko and mania are only time bassed, and don't have anything that even could give you statistics for a accuracy heatmap, they wouldn't have it (tho i think some things should be also shown for taiko and mania)
!ctb does require aim like in !std, exept only in an X axis, and not X and Y axises like in !std, but i think it would be cool to see your aim similary to !std.

**Comment by peppy** [comment]
Definitely sounds like something we want!

**Comment by felipemarins** [comment]
I don't know if you should select an answer for a discussion while there isn't an implementation for it yet, as the discussion can be used as a place for tracking it and for eventual contributors to check possible feature suggestions to implement.

**Reply by Tenexxt** [reply]
Alr

**Comment by xatatonik** [comment]
maybe one day 
<img width="500" height="541" alt="emoji-sad" src="https://github.com/user-attachments/assets/b4e82582-2e6f-4b49-af94-d0cdd4591771" />

**Reply by Tenexxt** [reply]
someday surely 😅
